### ISSUE TITLE
Tabs cannot be toggled using keyboard commands

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2716]
Found via Trusted Tester, tabs on the homepage and How Revenue Works page cannot be toggled using keyboard commands. 

![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/6a7c18d1-e023-4ba4-8602-51e966b5be12)

![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/77a91108-9ef3-4ccd-a0ae-104b96ce6843)

**Comment by lpgoldstein** [comment]
No longer relevant
